src/NewChain/AuxiliaryChain.ts

```diff
@@ -219,6 +219,25 @@ export default class AuxiliaryChain {
     return ostCoGateway.progressMint(messageHash, hashLockSecret, this.txOptions);
   }
 
+  /**
+   * Resets organization contracts admin address to 0x.
```

Comment:
`0x` meaning a zero length hex value?

```suggestion
   * Resets organization contracts admin address to `address(0)`.
```

Reply:
Done!